net/wol/src/opnsense/mvc/app/models/OPNsense/Wol/Wol.xml

```diff
@@ -10,6 +10,7 @@
             <default></default>
             <filters>
                 <enable>/^(?!0).*$/</enable>
+                <if>/^(?!openvpn|ipsec|enc).*$/i</if>
```

Comment:
@fabianfrz works, thanks! but I'll investigate if we could have a better fix to only include non-virtual ones as new ones could be added and this is static

Reply:
@fichtner maybe add a flag to the interface field type

Reply:
yupp, good idea... I'll look into it :)